test/stream-monitor/flogging/infra_logging.py

```diff
@@ -46,8 +46,23 @@ def filter(self, record):
         else:
             gname = getattr(cur, 'log_facility', str(cur))
         record.greenlet = '{0!s}'.format(gname)
-        return True
 
+        # This section removes the process, path, and greenlet information 
+        # from any mssage that starts with any of the characters ' +-'. 
+        # These are well known stream monitor messages and this informtion
+        # is not required to be displayed.
+        # TODO: there is probably a better way to do this but this will
+        #       work until it is setup.
+        if any(elem in record.msg[0] for elem in r' +-?%'):
+            record.location_info_string = ''
+        else:    
+            record.location_info_string = '{} {} {}:{}@{} {}'.format(record.process,
```

Comment:
Go ahead and remove 'original' from below, but move the "%(process)d  %(processName)s %(pathname)s:%(funcName)s@%(lineno)d %(greenlet)s'" here. Err, that might be clearer if I just showed it :):

`                record.location_info_string = '%(process)d  %(processName)s %(pathname)s:%(funcName)s@%(lineno)d %(greenlet)s'.format(record)`

That keeps the connection between the logging formatters and this statement much more clear, I think.